data/strategies/individual_strats/StrategyVaultUSDC.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "strategy",
+  "name": "v1 aLink Borrow Strategy",
+  "description": "This vault deposits {{token}} as collateral on Aave to borrow USDC. The USDC is deposited into the v1 USDC yVault. Profits are harvested and used to buy additional underlying tokens, supplied as collateral on Aave in exchange for {{token}} and re-deposited into the vault.",
```

Comment:
FIX

Reply:
Deleting since we are not having v1 strategies in this folder